.github/ISSUE_TEMPLATE/11-log-report.md

```diff
@@ -0,0 +1,43 @@
+---	
+name: Log Report - If citizen AI's has log issue
+about: Please follow the given template
+title: ''	
+labels: Bug, AI Log
+assignees: ''	
+
+---	
+
+<!-- PLEASE DO NOT DELETE TOPICS AS YOUR ISSUE WILL GET CLOSED -->
+### Prerequisites <!-- Replace spaces with "x" to tick boxes. [ ] => [x] -->
+
+- [ ] I am running the latest **alpha** version of MineColonies and Structurize for my Minecraft version.
```

Comment:
```suggestion
- [ ] I am running the latest **alpha** version of MineColonies for my Minecraft version.
```
Structurize isn't that relevant here, and isn't requested below

Reply:
is relevant but whatever